src/test/kotlin/com/zepben/evolve/services/network/testdata/FillFields.kt

```diff
@@ -667,6 +667,9 @@ fun EquivalentEquipment.fillFields(service: NetworkService, includeRuntime: Bool
 
 fun Analog.fillFields(service: NetworkService, includeRuntime: Boolean = true): Analog {
     (this as Measurement).fillFields(service, includeRuntime)
+
+    positiveFlowIn = true
+
     return this
 }
 
```

Comment:
Question: regarding the comment on `Switch.fillFIelds` (github won't let me comment on the exact lines sorry), do we plan on ever adding unganged switch state to the protobuf library?

Reply:
I would expect it to be added at some point, particularly if we get traction in the US where unganged is more common.